presto-hive-metastore/src/test/java/com/facebook/presto/hive/metastore/TestHiveMetastoreUtil.java

```diff
@@ -200,4 +201,15 @@ public void testReconstructPartitionSchema()
         assertThatThrownBy(() -> reconstructPartitionSchema(ImmutableList.of(c1), 2, ImmutableMap.of(0, c2)))
                 .isInstanceOf(IllegalArgumentException.class);
     }
+
+    @Test
+    public void testExtractPartitionValues()
+    {
+        String p1 = "date_str";
```

Comment:
shall we have a concrete example?
e.g.
datePartition = "20201221"
countryPartition = "united_states"

Reply:
Ok, let me give a concrete example